lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -122,7 +121,8 @@ def test_credit_requirements_eligible(self):
             response,
             "{}, you have met the requirements for credit in this course.".format(self.USER_FULL_NAME)
         )
-        self.assertContains(response, "Completed by {date}".format(date=self._now_formatted_date()))
+        self.assertContains(response, "Completed by {date}")
+        self.assertContains(response, datetime.datetime.now(UTC).strftime('%Y-%m-%d %H:%M'))
```

Comment:
Isn't there a race condition here?  What if there was a context-switch while running the test and it was delayed and the datetime.now() is not the same value in the test and when it was calculated in the code?